### ISSUE TITLE
你好

### ISSUE THREAD

**Issue by moondaiy** [JianshuZhang/RAN#2]
看了下关于RAN的paper, 请问,source code什么时候可以开源额. 想复现下这个网络,但是有一些细节实在把握不好,,还请作者可以尽快开源...

**Comment by JianshuZhang** [comment]
Sorry, out TPAMI paper is still under reviewed. We may not release our source codes until our paper is accepted : ).

**Comment by lugeluge** [comment]
Excuse me. Thank you very much for your paper. It's really great. When will the source code be opened? @JianshuZhang

**Comment by JianshuZhang** [comment]
@lugeluge Our journal paper is under reviewed by IEEE TPAMI, we are still waiting for the first review comment. If we have positive comments, the source codes will be released. Actually, the current released RAN contents are just for the review.

**Comment by dyz330542613** [comment]
Excuse me. I wonder if IEEE TPAMI has given you responses? When will you release the source code? 

Many thanks!

**Comment by JianshuZhang** [comment]
Sorry for now, our first review is not good, we may keep the source codes. We are under the second review.

**Comment by WeihongM** [comment]
Congratulations, I have seen your paper has been accepted by PR.
Do you plan to make code available? @JianshuZhang

**Comment by JianshuZhang** [comment]
Yes, we plan to release the pytorch code soon, but you can also choose to use the theano codes in the WAP project to run a test, since they are both encoder decoder. As for the data, we need some time to make them clear, as it has passed so long, I am a little confused now.

发自我的iPhone

**Comment by WeihongM** [comment]
Okay, looking forward to your update.

**Comment by phybrain** [comment]
champing at the bit

**Comment by xyl576807077** [comment]
Will the source code be opened soon ?

**Comment by Crescentz** [comment]
请问开源的时间？  另外ids文件里是不是好多错误啊

**Comment by Cocoalate** [comment]
同问开源时间~

**Comment by phybrain** [comment]
> 同问开源时间~

已经废了 效果不行

**Comment by Cocoalate** [comment]
> 同问开源时间~
> 
> 已经废了 效果不行

大佬你是复现了么？发现效果不行？

**Comment by JingyeChen** [comment]
您好，我复现了一下RAN，发现Table2的我跑出来的实验结果并没有文章高。您是有用到什么trick么（例如ensemble），但未在文章里提及。

**Comment by JingyeChen** [comment]
请问能开源一下代码么~方便进行公平的比较:D

**Comment by leviethung2103** [comment]
Given the dictionary, how can we construct the original word from sub parts ? Thank you

**Comment by Chen-stac** [comment]
@JianshuZhang  Really appreciate for your paper. When will the source code be opened?
